app/ember-copy.js

```diff
@@ -0,0 +1,2 @@
+export { default as copy } from 'ember-copy/copy';
```

Comment:
Shouldn't need this file at all (we don't want to export in the app directory)...